locale/de/LC_MESSAGES/django.po

```diff
@@ -766,6 +769,8 @@ msgid ""
 "            confirm that <a href=\"mailto:%(email)s\">%(email)s</a> is an\n"
 "            e-mail address for user %(user_display)s."
 msgstr ""
+"Bitte bestätigen Sie, dass <a href=\"mailto:%(email)s\">%(email)s</a> eine E-"
+"Mail Adresse des Nutzers %(user_display)s ist."
```

Comment:
"Nutzers" should be gendered.

Reply:
Note to myself: has to go back to meinBerlin, too